src/components/timeline/index.tsx

```diff
@@ -117,10 +119,15 @@ const Timeline = (props: TimelineProps) => {
   };
 
   return (
-    <View row style={containerStyle}>
-      <View style={styles.timelineContainer}>
+    <View row style={containerStyle} testID={testID}>
+      <View style={styles.timelineContainer} testID={`${testID}.timelineContainer`}>
```

Comment:
Why we need this testID as well?

Reply:
I thought that could be helpful in some cases if we will want to test the entire `timeline` container.
But since the `lines` and the `point` as their own test id's can remove it.